templates/vue/src/components/NodeModal.vue

```diff
@@ -77,16 +110,16 @@
         <b-tab title="Appearance">
           <div id="modal-appearance">
             <b-form-group>
-              <b-form-checkbox
-                v-model="addThumbnail"
-              >Add a thumbnail</b-form-checkbox>
+              <b-form-checkbox v-model="addThumbnail"
+                >Add a thumbnail</b-form-checkbox
+              >
```

Comment:
This looks really weird. Either put it all on one line or put "Add a thumbnail" on its own line with the opening tag and closing tag also on their own line.